colossalai/shardformer/modeling/bert.py

```diff
@@ -109,5 +111,24 @@ def bert_flash_attention_forward(
         if self.is_decoder:
             outputs = outputs + (past_key_value,)
         return outputs
-    
-    return bert_flash_attention_forward
\ No newline at end of file
+
+    return forward
+
+
+def _get_jit_fused_output_forward():
+
+    def forward(self, hidden_states: torch.Tensor, input_tensor: torch.Tensor) -> torch.Tensor:
+        hidden_states = self.dense(hidden_states)
+        hidden_states = self.dropout_add(hidden_states, input_tensor, self.dropout.p, self.dropout.training)
+        hidden_states = self.LayerNorm(hidden_states)
+        return hidden_states
+
+    return forward
+
+
+def get_jit_fused_bert_self_output_forward():
+    return _get_jit_fused_output_forward()
+
+
+def get_jit_fused_bert_output_forward():
```

Comment:
These two functions call the same function but their name are different. It's confusing.

Reply:
> Member

Updated.